### PR TITLE
Modify params to match perion's requirements

### LINKED ISSUE #1: Tweak the parameters to suit perion
- [x] two pools, one for LP on sushi swap and one for single token staking
- [x] variable naming
- [x] min 1 month lockup
- [x] max 3 years lockup 
- [x] reward multiplier up to 5x, formula: `1x + months / 9`
- [x] staking with sPERC and sPERCPOOL
- [x] User can claim rewards at any point

### PR DESCRIPTION
**Pull request by perionjames**
Closes #1

### CODE REVIEW

File: tasks/deploy/full.ts [hunk 1]
```diff
@@ -5,7 +5,8 @@ import sleep from "../../utils/sleep";
 import { constants, utils } from "ethers";
 import { captureRejectionSymbol } from "events";
 
-const MC = "0x949d48eca67b17269629c7194f4b727d4ef9e5d6";
+const PERC = "0x60bE1e1fE41c1370ADaF5d8e66f07Cf1C2Df2268";
+// TODO change addresses
 const LP = "0xcCb63225a7B19dcF66717e4d40C9A72B39331d61";
```
Comment (perionjames) [inline tasks/deploy/full.ts]: @amosperion any idea what these should be?

File: tasks/deploy/full.ts [hunk 2]
```diff
@@ -5,7 +5,8 @@ import sleep from "../../utils/sleep";
...
 const multisig = "0x7e9e4c0876b2102f33a1d82117cc73b7fddd0032";
```
Comment (perionjames) [inline tasks/deploy/full.ts]: @amosperion as above

Comment (amosperion) [reply tasks/deploy/full.ts]: 0x12D73beE50F0b9E06B35Fdef93E563C965796482

Comment (amosperion) [reply tasks/deploy/full.ts]: That's the Perion Multisig safe

File: tasks/deploy/full.ts (hunk 1 above)
Comment (amosperion) [reply tasks/deploy/full.ts]: Is LP where the current pool is?

Comment (amosperion) [reply tasks/deploy/full.ts]: 0x45b6ffb13e5206dafe2cc8780e4ddc0e32496265

**Comment by perionjames** [comment]
@amosperion @geoshak up for review, would like a few eyes on the formula...

**Review by amosperion — CHANGES_REQUESTED** [review]

**Comment by geoshak** [comment]
I'll run the tests locally in the future and share comments, if any.

**Pull request merged**